docs/resources/pipeline.md

```diff
@@ -108,7 +108,7 @@ resource "codefresh_pipeline" "test" {
 - `location` - (Optional) Default value - **git**.
 - `repo` - (Required) The GitHub `account/repo_name`.
 - `path` - (Required) The relative path to the Codefresh pipeline file.
-- `revison` - (Required) The git revision.
+- `revison` - (Optional) The git revision (e.g. branch). Leave blank to use the branch from the git trigger triggering the pipeline.
```

Comment:
@logicbomb421 pre-existing typo on the name of the attribute (`revison` instead of `revision`) in case you would like to amend it while editing this line ;)